dask/array/tests/test_array_core.py

```diff
@@ -97,6 +97,15 @@ def test_top():
     }
 
 
+def test_top_with_kwargs():
+    assert top(
+        add_with_default, "z", "i", "x", "i", numblocks={"x": (2, 0)}, b=100
+    ) == {
+        ("z", 0): (apply, add_with_default, [("x", 0)], {"b": 100}),
+        ("z", 1): (apply, add_with_default, [("x", 1)], {"b": 100}),
+    }
```

Comment:
Is `b` here supposed to be `y`?  Also, would `operator.add` work instead of add_with_default?

Reply:
Yup, using `operator.add` works here as well. PR was updated with this change.